smartconnect/__init__.py

```diff
@@ -440,10 +407,8 @@ def post_smart_request(self, *, json: str, ca_uuid: str = None):
         url = f'{self.api}/api/data/{ca_uuid}'
         response = self._session.post(url,
                                  headers={'content-type': 'application/json'},
-                                 data=json,
-                                 timeout=DEFAULT_TIMEOUT,
-                                 verify=self.verify_ssl)
-        if response.ok:
+                                 data=json)
```

Comment:
The `data` param in httpx is primarily for sending form data, although it works with json strings for now, but that will be deprecated in favor of setting `content` (you may be getting a deprecation warning already).
```suggestion
                                 content=json)
```